File: FitProjectWin/FitProjectWin/Services/FirebaseConfig.cs
namespace FitProjectWin.Services;

public static class FirebaseConfig
{
    public const string ApiKey = "AIzaSyDcR92CtVhDqtxeEGRu_8JGuGw2UicjQQ0";
    public const string ProjectId = "workouts-67e5d";
    public const string AuthBaseUrl = "https://identitytoolkit.googleapis.com/v1";
    public const string FirestoreBaseUrl = $"https://firestore.googleapis.com/v1/projects/{ProjectId}/databases/(default)/documents";
    public const string StorageBucket = "workouts-67e5d.appspot.com";
    public const string StorageBaseUrl = $"https://firebasestorage.googleapis.com/v0/b/{StorageBucket}/o";
}

File: FitProjectDroid/app/src/main/java/com/fitproject/droid/data/FirebaseConfig.kt
package com.fitproject.droid.data

/**
 * FitPros Firebase project constants (from app.fitpros.io web config).
 *
 * Android also reads [com.google.firebase:firebase] values from `app/google-services.json`.
 * The checked-in file uses the correct project number but a placeholder `mobilesdk_app_id`
 * until FitPros registers `com.fitproject.droid` in Firebase Console and provides the
 * official download. Replace `google-services.json` with that file when available.
 */
object FirebaseConfig {
    const val ApiKey = "AIzaSyDcR92CtVhDqtxeEGRu_8JGuGw2UicjQQ0"
    const val ProjectId = "workouts-67e5d"
    const val MessagingSenderId = "921130967436"
    const val StorageBucket = "workouts-67e5d.appspot.com"
    const val PackageName = "com.fitproject.droid"
}